generated/json_data/skippable_cutscenes.jsonc

```diff
@@ -4731,7 +4731,7 @@
                             "position": [
                                 799.513916,
                                 -168.868607,
-                                72.204781
+                                72.2047958
```

Comment:
Why change this?

Reply:
those .0001 units clearly make an enormous difference

Reply:
Sorry, these are changes from a previous PR, I don't know why they showed up here.